tests/test_formats.py

```diff
@@ -1489,6 +1489,415 @@ def test_fmt_image_path():
     assert 'src="/a/b/c"' in strip_windows_drive(res)
 
 
+def test_fmt_units():
```

Comment:
It's not clear what about fmt units is being tested here. Please...

* test values to be formatted one by one
* only test values that test / make clear something new about fmt_units